centos.repo

```diff
@@ -1,13 +1,13 @@
 [centos-8-base-os]
 name = CentOS - BaseOS
-baseurl = https://vault.centos.org/8.3.2011/BaseOS/x86_64/os/
+baseurl = http://mirror.centos.org/centos/8.4.2105/BaseOS/x86_64/os/
 enabled = 1
 gpgkey = https://www.centos.org/keys/RPM-GPG-KEY-CentOS-Official
 gpgcheck = 1
 
 [centos-8-appstream]
 name = CentOS - AppStream
-baseurl = https://vault.centos.org/8.3.2011/AppStream/x86_64/os/
+baseurl = http://mirror.centos.org/centos/8.4.2105/AppStream/x86_64/os/
```

Comment:
I suppose when Centos 8.4 is deprecated we'll need to switch this back to mirror.centos.org

Reply:
Also weird that mirror.centos.org isn't served over TLS

Reply:
That's a security issue.  We can't trust the server's identity over HTTP, and that leave us vulnerable to a supply chain attack.  We can't allow a non TLS server.

Reply:
The packages' integrity is checked with the Centos gpg key so we can trust the packages.

Server identity is another matter though. I suppose we could pick a mirror from: https://www.centos.org/download/mirrors/

Reply:
Got it, I'll update the files to use https mirrors. Thanks!